modules/dynamic-plug-in-api.adoc

```diff
@@ -1476,7 +1479,7 @@ Creates full page ErrorBoundaryFallbackPage component to display the "Oh no! Som
 [discrete]
 == `PerspectiveContext`
 
-@deprecated - use the provided `usePerspectiveContext` insteadCreates the perspective context.
+Deprecated: Use the provided `usePerspectiveContext` instead. Creates the perspective context.
```

Comment:
Do these `@deprecated` have a special meaning, or were they just to indicate that something was deprecated? Assumed the latter, and changed to just "Deprecated: "

Reply:
I don't think they have any particular meaning. I am good with "Deprecated:"